loan-pattern/README.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: pattern title: Loan-pattern folder: Loan-pattern permalink: /patterns/Loan-pattern/ categories: Behavioral
+language: en tags:
+
+- Decoupling
+
+---
```

Comment:
This looks a bit malformed. Please double check that it conforms with https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute